src/plugins/plan/components/Plan.vue

```diff
@@ -133,6 +133,9 @@ export default {
     this.clipActivityNames = this.configuration.clipActivityNames;
     if (this.domainObject.type === 'plan') {
       this.planData = getValidatedData(this.domainObject);
+      if (!Object.keys(this.planData).length) {
+        this.openmct.notifications.error('Error loading JSON. Please verify JSON is valid.');
```

Comment:
Does this still end up creating an invalid Plan object after creating the error notification? One way we could prevent users from creating an invalid Plan object is to do this validation as part of the create form validation, using some kind of JSON schema validator.

Reply:
yes? it's an empty object. I'll look into what would be involved with JSON schema validator before hand

Reply:
I didn't know how how to handle valid JSON but invalid plan json. For example, I can open valid JSON:
```
{
  "token": {
    "colorPrimary": "#da0ffb",
    "wireframe": false
  }
}
``` 
and this also fails. Having notification at this point catches this case as well.

Reply:
have updated to handle both cases, invalid json won't load, and invalid schema json will show error